File: src/python/Messages/SynapSysResponse.py
from dataclasses import dataclass, field
from typing import Any, Optional

@dataclass
class SynapSysResponse:
    ok: bool
    provider: str
    model: str
    requestId: Optional[str] = None
    output: Optional[str] = None
    raw: Optional[dict] = field(default_factory=dict)
    error: Optional[dict] = field(default_factory=dict)

File: src/java/Messages/SynapSysResponse.java
package java.Messages;

import java.util.Map;

public class SynapSysResponse {
    public boolean ok;
    public String provider;
    public String model;
    public String requestId;
    public String output;
    public Map<String, Object> raw;
    public Map<String, Object> error;

    public SynapSysResponse() {}
}